### PR TITLE
Add Deployment Helpers and PrizePoolNetwork Update Scripts

### PR DESCRIPTION
**Pull request by kamescg**
Start preparing the mainnet deployment scripts and upgrade path.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,42 +9,47 @@
...
+    "deploy-mainnet-to-polygon": "DEPLOY=polygon hardhat deploy --network polygon",
+    "deploy-mainnet-to-ethereum": "DEPLOY=mainnet hardhat deploy --network mainnet",
+    "deploy:mainnet": "DEPLOY=mainnet hardhat deploy --network localhost",
+    "deploy:polygon": "DEPLOY=polygon hardhat deploy --network localhost",
+    "fork:mainnet": "FORK_CHAIN_ID=$MAINNET_CHAIN_ID FORK_RPC_URL=$MAINNET_RPC_URL FORK_ENABLED=true DEPLOY=mainnet hardhat node --no-deploy --network hardhat",
+    "fork:polygon": "FORK_CHAIN_ID=$POLYGON_CHAIN_ID FORK_RPC_URL=$POLYGON_RPC_URL FORK_ENABLED=true DEPLOY=polygon hardhat node --no-deploy --network hardhat",
+    "node:ethereum": "DEPLOY=ethereumMainnet hardhat node --network hardhat",
+    "node:polygon": "DEPLOY=polygonMainnet hardhat node --network hardhat",
+    "node:avalanche": "DEPLOY=avalancheMainnet hardhat node --network hardhat",
+    "upgrade:mainnet": "DEPLOY=mainnet.upgrade.v1.10.0 hardhat deploy --network mainnet",
```
Comment (aodhgan) [inline package.json:29]: is the minor version `0.10` or `0.1`?

Comment (kamescg) [reply package.json]: You're right @aodhgan it should be `v1.1.0`.

File: package.json [hunk 2]
```diff
@@ -9,42 +9,47 @@
...
+    "upgrade:polygon": "DEPLOY=polygon.upgrade.v1.10.0 hardhat deploy --network polygon",
+    "upgrade:mainnet:test": "DEPLOY=mainnet.upgrade.v1.10.0 hardhat deploy --network localhost",
+    "upgrade:polygon:test": "DEPLOY=polygon.upgrade.v1.10.0 hardhat deploy --network localhost",
+    "verify:mainnet": "hardhat --config hardhat.config.ts run ./scripts/verify.ts --network mainnet",
+    "verify:polygon": "hardhat --config hardhat.config.polygon.ts run ./scripts/verify.ts --network polygon",
+    "verify:avalanche": "hardhat --config hardhat.config.avlanche.ts run ./scripts/verify.ts --network avalanche"
```
[the remark this answers is not in the document]
Comment (kamescg) [reply package.json]: Yes - typo. Fixed.

**Comment by asselstine** [comment]
There are some issues let's talk about, including:

- The deployment scripts have gone from 2 files to 20+ files.
- instead of using the `src` directory, a separate `helper` directory was created
- modules from from `src` directory were copied into the helper directory.  Specifically `colors` and `transferOwnership`.  The file `deployAndLog` was copied and slightly altered to be `deployContract`
- a temporary file `populatedTransactions.beacon.1.json` is created in the `deploy/v1.1.0` during runtime and then was committed to git.
- modules have nearly identical names: `handleBeaconChainContractDeploy` vs `handleBeaconDeployment`

**Comment by asselstine** [comment]
Closing in favour of the other PR

**Pull request closed without merging**